kratos/includes/kratos_parameters.h

```diff
@@ -538,6 +538,12 @@ class KRATOS_API(KRATOS_CORE) Parameters
      */
     std::string GetString() const;
 
+    /**
+     * @brief This method returns the array of strings in the current Parameter
+     * @return The array of strings
+     */
+    std::vector<std::string> GetStringArray() const;
```

Comment:
also `GetStringVector` would IMO be a better fit, since it
- is consistent with the data-type
- is also consistent with the already existing `GetVector` function

Reply:
At first I also thought it would be more appropriate since we return `std::vector`. But finally I decided against it since `Parameters::IsVector()` would return false for string values. Calling it `GetStringVector()` would make the meaning of `Vector` in the `Parameters` ambiguous and confusing.

Reply:
ok